src/config/schema.types.d.ts

```diff
@@ -16,6 +16,16 @@ type AccountLimitsConfig = {
   }
 }
 
+type CustomField = {
+  name: string
+  type?: "string" | "integer" | "float" | "boolean"
```

Comment:
I think this should be mandatory

Reply:
type is not required, was an issue in schema validator https://github.com/GaloyMoney/galoy/pull/1465/commits/c77987b90c2efa222f03c8aa24b6e792af811ba2 also a unit test was added previously to validate default type https://github.com/GaloyMoney/galoy/pull/1465/files#diff-f2dc8f7e9dae5e09bcaebb4926c4fac76f9ec33907320097ae665330c04aa1afR27

Reply:
I don't understand - why is this optional?

Reply:
default is string, only name is required